### PR TITLE
Update the link for new Discussion

### PR DESCRIPTION
**Pull request by aiwithwhitehat**
## What are you changing in this pull request and why?
Based on the discussion here: https://github.com/dbt-labs/docs.getdbt.com/discussions/2553
The discourse will be the new forum where one can discuss new ideas.

This PR replaces the previous [link](https://github.com/dbt-labs/docs.getdbt.com/discussions) to start a new discussion to the [discourse link](https://discourse.getdbt.com/)

### CODE REVIEW

**Comment by aiwithwhitehat** [comment]
It says there is 1 unresolved Comment. How I can access my Versel account as it is pending approval for access?

**Comment by aiwithwhitehat** [comment]
@mirnawong1: Can you please review?

File: website/docs/community/resources/oss-expectations.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "Expectations for OSS contributors"
...
 
-Have you seen things in other OSS projects that you quite like, and think we could learn from? [Open a discussion on the Developer Hub](https://github.com/dbt-labs/docs.getdbt.com/discussions/new), or start a conversation in the dbt Community Slack (for example: `#community-strategy`, `#dbt-core-development`, `#package-ecosystem`, `#adapter-ecosystem`). We always appreciate hearing from you!
+Have you seen things in other OSS projects that you quite like, and think we could learn from? [Open a discussion on the DBT Community Forum](https://discourse.getdbt.com), or start a conversation in the dbt Community Slack (for example: `#community-strategy`, `#dbt-core-development`, `#package-ecosystem`, `#adapter-ecosystem`). We always appreciate hearing from you!
```
Comment (mirnawong1) [inline website/docs/community/resources/oss-expectations.md]: changing dbt to lowercase
```suggestion
Have you seen things in other OSS projects that you quite like, and think we could learn from? [Open a discussion on the dbt Community Forum](https://discourse.getdbt.com), or start a conversation in the dbt Community Slack (for example: `#community-strategy`, `#dbt-core-development`, `#package-ecosystem`, `#adapter-ecosystem`). We always appreciate hearing from you!
```

File: website/docs/terms/view.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ displayText: view
 hoverSnippet: A view (as opposed to a table) is a defined passthrough SQL query that can be run against a database (or data warehouse).
 ---
 :::important This page could use some love
-This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the docs.getdbt.com GitHub repository](https://github.com/dbt-labs/docs.getdbt.com/discussions) to begin the process of becoming a glossary contributor!
+This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the DBT Community Forum](https://discourse.getdbt.com) to begin the process of becoming a glossary contributor!
```
Comment (mirnawong1) [inline website/docs/terms/view.md]:
```suggestion
This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the dbt Community Forum](https://discourse.getdbt.com) to begin the process of becoming a glossary contributor!
```

File: website/docs/terms/table.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ displayText:  table
 hoverSnippet: In simplest terms, a table is the direct storage of data in rows and columns.  Think excel sheet with raw values in each of the cells.  
 ---
 :::important This page could use some love
-This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the docs.getdbt.com GitHub repository](https://github.com/dbt-labs/docs.getdbt.com/discussions) to begin the process of becoming a glossary contributor!
+This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the DBT Community Forum](https://discourse.getdbt.com) to begin the process of becoming a glossary contributor!
```
Comment (mirnawong1) [inline website/docs/terms/table.md]:
```suggestion
This term would benefit from additional depth and examples. Have knowledge to contribute? [Create an issue in the docs.getdbt.com repository](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose) to begin the process of becoming a glossary contributor!
```

File: website/docs/terms/view.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ displayText: view
 hoverSnippet: A view (as opposed to a table) is a defined passthrough SQL query that can be run against a database (or data warehouse).
 ---
 :::important This page could use some love
-This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the docs.getdbt.com GitHub repository](https://github.com/dbt-labs/docs.getdbt.com/discussions) to begin the process of becoming a glossary contributor!
+This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the dbt Community Forum](https://discourse.getdbt.com) to begin the process of becoming a glossary contributor!
```
Comment (mirnawong1) [inline website/docs/terms/view.md]:
```suggestion
This term would benefit from additional depth and examples. Have knowledge to contribute? [Create an issue in the docs.getdbt.com repository](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose) to begin the process of becoming a glossary contributor!
```

File: website/docs/terms/materialization.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ hoverSnippet: The exact Data Definition Language (DDL) that dbt will use when cr
 </head>
 
 :::important This page could use some love
-This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the docs.getdbt.com GitHub repository](https://github.com/dbt-labs/docs.getdbt.com/discussions) to begin the process of becoming a glossary contributor!
+This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the DBT Community Forum](https://discourse.getdbt.com) to begin the process of becoming a glossary contributor!
```
Comment (mirnawong1) [inline website/docs/terms/materialization.md]:
```suggestion
This term would benefit from additional depth and examples. Have knowledge to contribute? [Create a discussion in the DBT Community Forum](https://discourse.getdbt.com) to begin the process of becoming a glossary contributor!
```
```suggestion
This term would benefit from additional depth and examples. Have knowledge to contribute? [Create an issue in the docs.getdbt.com repository](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose) to begin the process of becoming a glossary contributor!
```

File: website/docs/community/resources/oss-expectations.md [hunk 6]
```diff
@@ -51,7 +51,7 @@ An issue could be a bug you’ve identified while using the product or reading t
 
 ### Best practices for issues
 
-- Issues are **not** for support / troubleshooting / debugging help. Please [open a discussion on the Developer Hub](https://github.com/dbt-labs/docs.getdbt.com/discussions/new), so other future users can find and read proposed solutions. If you need help formulating your question, you can post in the `#advice-dbt-help` channel in the [dbt Community Slack](https://www.getdbt.com/community/).
+- Issues are **not** for support / troubleshooting / debugging help. Please [open a discussion on the DBT Community Forum](https://discourse.getdbt.com), so other future users can find and read proposed solutions. If you need help formulating your question, you can post in the `#advice-dbt-help` channel in the [dbt Community Slack](https://www.getdbt.com/community/).
```
Comment (mirnawong1) [inline website/docs/community/resources/oss-expectations.md]:
```suggestion
- Issues are **not** for support / troubleshooting / debugging help. Please [open a discussion on the dbt Community Forum](https://discourse.getdbt.com), so other future users can find and read proposed solutions. If you need help formulating your question, you can post in the `#advice-dbt-help` channel in the [dbt Community Slack](https://www.getdbt.com/community/).
```

File: README.md [hunk 7]
```diff
@@ -17,7 +17,7 @@ Creating an inclusive and equitable environment for our documents is more import
 We welcome contributions from community members to this repo:
 - **Fixes**: When you notice an error, you can use the `Edit this page` button at the bottom of each page to suggest a change.
 - **New documentation**: If you contributed code in [dbt-core](https://github.com/dbt-labs/dbt-core), we encourage you to also write the docs here! Please reach out in the dbt community if you need help finding a place for these docs.
-- **Major rewrites**: You can [file an issue](https://github.com/dbt-labs/docs.getdbt.com/issues/new?assignees=&labels=content%2Cimprovement&template=improve-docs.yml) or [start a discussion](https://github.com/dbt-labs/docs.getdbt.com/discussions) to propose ideas for a content area that requires attention.
+- **Major rewrites**: You can [file an issue](https://github.com/dbt-labs/docs.getdbt.com/issues/new?assignees=&labels=content%2Cimprovement&template=improve-docs.yml) or [start a discussion](https://discourse.getdbt.com) to propose ideas for a content area that requires attention.
```
Comment (mirnawong1) [inline README.md]: for dbt product docs feedback/issues/suggestions, we ask users to open an issue in the docs.getdbt.com repo. for developer hub-related discussions (debugging help, sharing data practices, etc.), we recommend the discourse developer hub.
```suggestion
- **Major rewrites**: You can [file an issue](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose) to propose ideas for a content area that requires attention.
```

File: .github/ISSUE_TEMPLATE/improve-the-site.yml [hunk 8]
```diff
@@ -5,7 +5,7 @@ body:
   - type: markdown
     attributes:
       value: |
-        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/discussions)
+        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://discourse.getdbt.com)
```
Comment (mirnawong1) [inline .github/ISSUE_TEMPLATE/improve-the-site.yml]:
```suggestion
        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose)
```

File: .github/ISSUE_TEMPLATE/a-improve-docs.yml [hunk 9]
```diff
@@ -5,7 +5,7 @@ body:
   - type: markdown
     attributes:
       value: |
-        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/discussions)
+        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://discourse.getdbt.com)
```
Comment (mirnawong1) [inline .github/ISSUE_TEMPLATE/a-improve-docs.yml]:
```suggestion
        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose)
```

File: .github/ISSUE_TEMPLATE/a-improve-docs.yml [hunk 10]
```diff
@@ -5,7 +5,7 @@ body:
   - type: markdown
     attributes:
       value: |
-        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/discussions)
+        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose)
```
Comment (mirnawong1) [inline .github/ISSUE_TEMPLATE/a-improve-docs.yml]:
```suggestion
        * You can ask questions or submit ideas for the dbt docs in [Issues](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose)
```

File: .github/ISSUE_TEMPLATE/improve-the-site.yml [hunk 11]
```diff
@@ -5,7 +5,7 @@ body:
   - type: markdown
     attributes:
       value: |
-        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/discussions)
+        * You can ask questions or submit ideas for the dbt docs in [Discussions](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose)
```
Comment (mirnawong1) [inline .github/ISSUE_TEMPLATE/improve-the-site.yml]:
```suggestion
        * You can ask questions or submit ideas for the dbt docs in [Issues](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose)
```

**Review by mirnawong1 — APPROVED** [review]
hey @anksu2024, thanks so much for opening this pr and great work here! I've reviewed it and made some small tweaks and wanted to clarify a few things as well:

- any questions on solutions, sharing best practices, etc. -- we recommend users open up discussions in the dbt community forum in discourse. this would 
- any feedback, questions, requests, etc. for the dbt product docs (docs.getdbt.com site) -- we recommend users open an issue in the github repository. 

i wanted to clarify that the developer hub is all of https://docs.getdbt.com/ site, which includes the product docs, developer blog, guides, and so on. the dbt community forum is a place where you can get help and exchange knowledge with other data practitioners.

there were some places in the suggested changes that i updated to point to the docs.getdbt.com issues. it previously directed users to 'discussions' which wasn't correct either! i hope that clarifies it and i appreciate the awesome attention to detail in this pr. let me know if you have any questions at all!

I'll go ahead and merge this pr for you ✨

**Comment by aiwithwhitehat** [comment]
Thanks @mirnawong1 :)

**Pull request merged**